rust/relay/server/src/main.rs

```diff
@@ -586,6 +586,12 @@ where
                         ClientSocket::new(from),
                         Instant::now(),
                     ) {
+                        // If eBPF offloading is enabled, drop ChannelData relay traffic (it's handled by kernel)
```

Comment:
This comment is redundant with the code, no?

Reply:
Yeah I can remove it

Reply:
Done